src/app/page.tsx

```diff
@@ -46,16 +51,8 @@ export default function Home() {
           <div className="bg-blue-950 rounded-full size-12 flex justify-center items-center cursor-pointer text-white hover:bg-blue-900 duration-150 ">
             <IoCalendarOutline size={32} />
           </div>
```

Comment:
Agora que temos esse componente podemos otimizar essa parte, passar `<></>` no content seria o mesmo que dizer que ta passando um html "nulo",  quando a pessoa que implementar o calendário só vai precisar substituir pelo componente.

```suggestion
          <HomeSideButton icon={<IoCalendarOutline/>}  content={<></>}>
```